6/README.md

```diff
@@ -0,0 +1,1078 @@
+
+A limited set of the database functionality to support Tango Device Server configuration shall be avaliable as a read-only file.
+This is an option for use in the absence of a network service.
+
+The database SHALL enforce uniqueness of device names and aliases.
+
+The Database SHALL enforce the naming convention defined in all Tango RFCs
+for device name, the command name, the attribute name, the property name, the device alias name and the device server name.
+
+The device name, its domain, member and family fields and its alias maximum sizes SHALL BE at least:
+- device name: 255
+- domain field: 85
+- family field: 85
+- member field: 85
+- device alias name: 255
+
+## List of Commands Grouped by Functionality
+
+### Devices
+
+The database SHALL implement the following device-related commands:
+- DBAddDeviceDbDeleteDevice
+- DbGetClassForDevice
+- DbGetClassInheritanceForDevice
+- DbGetDeviceClassList
+- DbGetDeviceDomainList
+- DbGetDeviceFamilyList
+- DbGetDeviceInfo
+- DbGetDeviceList
+- DbGetDeviceWideList
+- DbGetDeviceMemberList
+- DbGetDeviceServerClassList
+
+### Classes
+The database SHALL provide a means of querying device server classes.
+
+The database SHALL implement the following class-related commands:
+- DbGetClassList
+- DbGetDeviceClassList
+- DbGetDeviceServerClassList
+
+
+### Properties
+The database SHALL provide a means of setting and getting properties for classes, devices, attributes, pipes and free objects.
+
+The database SHALL provide a means to retrieve the history of property value settings.
+
+The database SHALL implement the following property-related commands:
+
+##### ClassAttributeProperty
+- DbDeleteClassAttributeProperty
+- DbGetClassAttributeProperty
+- DbGetClassAttributeProperty2
+- DbGetClassAttributePropertyHist
+- DbPutClassAttributeProperty
+- DbPutClassAttributeProperty2
+
+##### ClassProperty
+- DbDeleteClassProperty
+- DbGetClassProperty
+- DbGetClassPropertyHist
+- DbGetClassPropertyList
+- DbPutClassProperty
+
+##### DeviceAttributeProperty
+- DbDeleteDeviceAttributeProperty
+- DbGetDeviceAttributeProperty
+- DbGetDeviceAttributeProperty2
+- DbGetDeviceAttributePropertyHist
+- DbPutDeviceAttributeProperty
+- DbPutDeviceAttributeProperty2
+- DbDeleteAllDeviceAttributeProperty
+
+##### DeviceProperty
+- DbDeleteDeviceProperty
+- DbGetDeviceProperty
+- DbGetDevicePropertyHist
+- DbGetDevicePropertyList
+- DbPutDeviceProperty
+
+##### Free Object Property
+- DbDeleteProperty
+- DbGetProperty
+- DbGetPropertyHist
+- DbGetPropertyList
+- DbPutProperty
+
+##### ClassPipeProperty
+- DbGetClassPipeProperty
+- DbDeleteClassPipeProperty
+- DbPutClassPipeProperty
+- DbGetClassPipePropertyHist
+
+##### DevicePipeProperty
+- DbGetDevicePipeProperty
+- DbDeleteDevicePipeProperty
+- DbDeleteAllDevicePipeProperty
+- DbPutDevicePipeProperty
+- DbGetDevicePipePropertyHist
+
+
+### Attributes
+The database SHALL provide a means of setting and getting attribute values for a class or device.
```

Comment:
'The database SHALL provide a means of setting and getting information about attributes for a class or device.'
The word `value` suggests that the database is an interface to devices.